src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
copy/paste of the main one with a few extra lines. I tried to import the main one using include resource but didn't have much luck